### PR TITLE
Fix #12764 - Update troubleshooting-kubeadm doc

### LINKED ISSUE #12764: Improvement for k8s.io/docs/setup/independent/troubleshooting-kubeadm/
**This is a...** 

- [ X] Feature Request
- [ ] Bug Report

**Problem:**
missing command for fixing

**Proposed Solution:**
please add missing command fixing issue "TLS certificate errors"
export KUBECONFIG=/etc/kubernetes/admin.conf

**Page to Update:**
https://kubernetes.io/docs/setup/independent/troubleshooting-kubeadm/#tls-certificate-errors

### PR DESCRIPTION
**Pull request by daminisatya**
#12764 

@neolit123

### CODE REVIEW

File: content/en/docs/setup/independent/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -155,6 +155,18 @@ Unable to connect to the server: x509: certificate signed by unknown authority (
   regenerate a certificate if necessary. The certificates in a kubeconfig file
   are base64 encoded. The `base64 -d` command can be used to decode the certificate
   and `openssl x509 -text -noout` can be used for viewing the certificate information.
+- Verify that the env variable `KUBECONFIG` is set to `NULL` using:
```
Comment (neolit123) [inline content/en/docs/setup/independent/troubleshooting-kubeadm.md]: > Unset the the `KUBECONFIG` environment variable using:

File: content/en/docs/setup/independent/troubleshooting-kubeadm.md [hunk 2]
```diff
@@ -155,6 +155,18 @@ Unable to connect to the server: x509: certificate signed by unknown authority (
...
+
+  ```sh
+  unset KUBECONFIG
+  ```
+
+  Or set it to the correct config file using:
```
Comment (neolit123) [inline content/en/docs/setup/independent/troubleshooting-kubeadm.md]: > Or set it to the default kubeconfig location:

**Comment by daminisatya** [comment]
@neolit123 Thank you for the review! I updated the PR with the suggestions

**Comment by Rajakavitha1** [comment]
Checked the updates on: https://deploy-preview-12825--kubernetes-io-master-staging.netlify.com/docs/setup/independent/troubleshooting-kubeadm/

/approve

**Pull request merged**